Add chart of number of tests performed per day.

annapowellsmith opened the pull request:
(**Note:** Data stubbed manually as not yet present in official endpoint. The endpoint will need to be updated before this PR can be merged.)

**Rationale for PR**

It would be useful for testing information to be added to the official dashboard. This PR adds a chart of the number of tests performed per day.

<img width="590" alt="Screenshot 2020-04-24 at 09 34 50" src="https://user-images.githubusercontent.com/78156/80192292-dd60e380-860e-11ea-97c2-5bffb8d2e259.png">

**Further note on data**

The data I have used is taken from DHSC's [daily Twitter reports](https://twitter.com/search?q=%22tests%20on%22%20(from%3ADHSCgovuk)&src=typed_query&f=live), which are also subject to small [post hoc amendments](https://www.gov.uk/guidance/coronavirus-covid-19-information-for-the-public#number-of-cases-and-deaths), so the numbers in the stub may not match final numbers precisely. Hopefully DHSC can easily make a time series available. 

The DHSC numbers before 21 April exclude Northern Ireland. This is noted on the About page.

xenatisch commented:
@paulfryers This MOP contribution is in line with our previous discussion. It's ready to go, but we don't have it in our prioritised backlog. 

@Ha-san-ali Could we please have a ticket associated with this as "under active dev"?

xenatisch commented on src/pages/About/About.js:
```diff
@@ -186,14 +186,22 @@ const About: ComponentType<Props> = ({ }: Props) => {
                 occurred. They can be downloaded as a csv file.
             </p>
-            <hr className={"govuk-section-break govuk-section-break--l"} />
+
+            <Styles.SectionHeader className={"govuk-heading-m"}>
+                Tests over time
+            </Styles.SectionHeader>
+            <p className={"govuk-body"}>
+                Test data is reported by DHSC. Tests are shown in charts and tables according to the day on which they were
+                carried out. Data earlier than 21 April excludes tests in Northern Ireland.
+            </p>
+            <hr className={"govuk-section-break govuk-section-break--l"}/>
```
Modifications to the wording of this section requires explicit approval from @paulfryers

xenatisch commented on src/pages/About/About.js:
```diff
@@ -186,14 +186,22 @@ const About: ComponentType<Props> = ({ }: Props) => {
 
             <p className={"govuk-body-s"}>
-                Updated 6pm 16 April 2020
+                Updated 6pm 23 April 2020
```
Must be updated at time the branch is merged.

xenatisch commented on src/pages/Regional/Regional.js:
```diff
@@ -67,7 +67,8 @@ const Regional: ComponentType<Props> = ({ }: Props) => {
     totalDeaths: 'Total number of COVID-19 associated UK deaths in hospital by date reported',
-    dailyDeaths: 'Daily number of COVID-19 associated UK deaths in hospital by date reported'
+    dailyDeaths: 'Daily number of COVID-19 associated UK deaths in hospital by date reported',
+    dailyTests: 'Daily number of tests in UK by date performed'
```
Wording requires approval by @paulfryers

Ha-san-ali commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Feature: Publish test numbers](https://trello.com/c/ILsHWjSs/286-feature-publish-test-numbers)

annapowellsmith commented:
I've just rebased to fix conflicts and force-pushed to c3ade810b75dc7e0a158a6dce59d2625b1b08340, and added a fixup commit 96ebaa82581f5adb3c8ad7e420fa0c2ec86a8b3e with newer stub data. Would be great to get this merged? 🙏

Ha-san-ali commented:
Hi @annapowellsmith, we do not currently have permission to publish this feature. Apologies for the holdup but until we get the appropriate approvals this valuable addition remains blocked.

Ha-san-ali commented:
We have made the decision to add testing data in a future sprint, however we will be not be reporting it in the way of this PR. @annapowellsmith we appreciate the contribution however we will not be moving forward with this design.

annapowellsmith commented:
Hi @Ha-san-ali any update on when testing data might be added?
